src/Helpers/UiHelpers.cs

```diff
@@ -113,10 +109,66 @@ public static UICheckBox CreateCheckBox(this UIComponent uiComponent, string tex
+        {
+            UIDropDown dropDown = uiComponent.AddUIComponent<UIDropDown>();
+
+            dropDown.atlas = GetAtlas("Ingame");
+            dropDown.position = position;
+            dropDown.size = new Vector2(150f, 30f);
+            dropDown.listBackground = "GenericPanelLight";
+            dropDown.itemHeight = 30;
+            dropDown.itemHover = "ListItemHover";
+            dropDown.itemHighlight = "ListItemHighlight";
+            dropDown.normalBgSprite = "ButtonMenu";
+            dropDown.disabledBgSprite = "ButtonMenuDisabled";
+            dropDown.hoveredBgSprite = "ButtonMenuHovered";
+            dropDown.focusedBgSprite = "ButtonMenu";
+            dropDown.listWidth = 150;
+            dropDown.listHeight = 500;
+            dropDown.foregroundSpriteMode = UIForegroundSpriteMode.Stretch;
+            dropDown.popupColor = new Color32(45, 52, 61, 255);
+            dropDown.popupTextColor = new Color32(170, 170, 170, 255);
+            dropDown.zOrder = 1;
+            dropDown.textScale = 0.8f;
+            dropDown.verticalAlignment = UIVerticalAlignment.Middle;
+            dropDown.horizontalAlignment = UIHorizontalAlignment.Left;
+            dropDown.selectedIndex = 0;
+            dropDown.textFieldPadding = new RectOffset(8, 0, 8, 0);
+            dropDown.itemPadding = new RectOffset(14, 0, 8, 0);
+            dropDown.items = items;
+
+            UIButton button = dropDown.AddUIComponent<UIButton>();
+            dropDown.triggerButton = button;
+            button.atlas = GetAtlas("Ingame");
+            button.text = "";
+            button.size = dropDown.size;
+            button.relativePosition = new Vector3(0f, 0f);
+            button.textVerticalAlignment = UIVerticalAlignment.Middle;
+            button.textHorizontalAlignment = UIHorizontalAlignment.Left;
+            button.normalFgSprite = "IconDownArrow";
+            button.hoveredFgSprite = "IconDownArrowHovered";
+            button.pressedFgSprite = "IconDownArrowPressed";
+            button.focusedFgSprite = "IconDownArrowFocused";
+            button.disabledFgSprite = "IconDownArrowDisabled";
+            button.foregroundSpriteMode = UIForegroundSpriteMode.Fill;
+            button.horizontalAlignment = UIHorizontalAlignment.Right;
+            button.verticalAlignment = UIVerticalAlignment.Middle;
+            button.zOrder = 0;
+            button.textScale = 0.8f;
+
+            dropDown.eventSizeChanged += new PropertyChangedEventHandler<Vector2>((c, t) =>
+            {
+                button.size = t; dropDown.listWidth = (int)t.x;
+            });
+
+            return dropDown;
+        }
+
+        // Sourced from : https://github.com/SamsamTS/CS-MoveIt/blob/master/MoveIt/GUI/UIUtils.cs
         // I found his code after I started this class, the below atlas feature is quite neat!
 
-        private static Dictionary<string, UITextureAtlas> _atlases;
+        private static Dictionary<string, UITextureAtlas>_atlases;
```

Comment:
Yeah the spaces need to all go back. I did try take a short cut with the translation but it went in the whole entire solution therefore when this commit was pushed I was going to rebase and start all the other commits again since I'm around 26 commits ahead of the master.